Update repo with '22 program changes (#21)
protocol/research-grants: README.md

# Protocol Labs Research Grants

## Overview

The Protocol Labs Research Grant Program aims to both support collaborative work on problems defined by the broader research community and drive progress on clearly scoped research projects critical to our work at Protocol Labs. We encourage proposals aligned with PL's mission to drive breakthroughs in computing to push humanity forward, and we welcome unique perspectives and diverse backgrounds.

## Funding instruments

Protocol Labs Research funds three broad categories of grants:
1. Open research grants, intended to support researchers pursuing topics of interest to Protocol Labs. These include:

>     * PhD Fellowships

Author: ```suggestion
    * Doctoral Fellowships
```

    * Doctoral Fellowships
    * Postdoctoral Fellowships
    * Faculty Summer Research Grants
    * Faculty Research Sabbatical Awards
3. A wildcard Request for Proposals, RFP-X, by which we fund research proposed by applicants where funding from one of our other grant structures does not readily apply. Note that we have a strong preference for funding via the open grants framework, and **RFP-X applications that could be handled within the existing framework will be rejected without review.**

Evaluation of applications for the open grants will occur in quarterly rounds. Decisions will normally be made and communicated to applicants within eight weeks of these submission deadlines. Evaluation for RFPs takes place according to the timeline in the call. RFP-X applications are evaluated year-round but usually have longer decision times.

> If you have any questions, please email us at **research@protocol.ai**.

Reviewer: Is this preferred over the research acceleration address?

Author: We generally avoid disclosing the internal team groups. That said, research-grants@ could be a better option?

Author: ```suggestion
If you have any questions, please email us at **research-grants@protocol.ai**.
```

### Open Research Grants

Applications for open grants can be submitted at any time and are processed in quarterly blocks. The cut-offs for each quarter are:
  * 15 February
  * 15 May
  * 15 August
  * 15 November

#### Protocol Labs PhD Fellowship

> The PhD Fellowship is intended to support the development of an early-stage researcher conducting research in one of PL's fields of interest. This award is available to Ph.D. candidates who have completed their MSc degree or have otherwise satisfied the requirements for PhD candidacy.

Author: ```suggestion
The Doctoral Fellowship is intended to support the development of an early-stage researcher conducting research in one of PL's fields of interest. This award is available to doctoral candidates who have completed their Master's degree or have otherwise satisfied the requirements for candidacy.
```

Author: @protocol/research-acceleration ^^ suggestions really are your friends

I think that's a good idea, but then I think we should do the same for PhD

> The **PL Research PhD Fellowship** consists of a fixed award of **$60,000**, intended to cover tuition and subsistence costs for a period of one year. PhD Fellowships can be renewed for a second year, by mutual agreement between PL and the grantee.

Author: ```suggestion
The **PL Research Doctoral Fellowship** consists of a fixed award of **$60,000**, intended to cover tuition and subsistence costs for a period of one year. Doctoral Fellowships can be renewed for a second year, by mutual agreement between PL and the grantee.
```

> This application may be submitted by the PhD candidate, their PI, or an administrative staff member from the host  institution. The submitter will serve as the primary point of contact for all communications regarding this application.

Author: ```suggestion
This application may be submitted by the doctoral candidate, their PI, or an administrative staff member from the host institution. The submitter will serve as the primary point of contact for all communications regarding this application.
```
